src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -611,6 +611,23 @@
             "additionalProperties": false,
             "properties": {
                 "develop": {"type": "boolean", "default": false},
```

Comment:
You can't have the key present twice. The `if` on the next line should be an if-else. Alternatively, you can put the `if` only around the one branch of the `oneOf` (so that for < 3.2 you get only the boolean branch), which would avoid the need to repeat this line.

Reply:
I've changed this to put the `if` around the one branch of the `oneOf`